package.json

```
@@ -146,6 +146,7 @@
       "apps/*",
       "libs/*",
       "packages/*",
+      "packages-enterprise/*",
```

```suggestion
      "enterprise/*",
```

@LetItRock what do you think for jsut calling this `enterprise` as the folder?